storage/innobase/xtrabackup/test/t/remote_tablespaces.sh

```diff
@@ -15,16 +15,28 @@ mkdir -p $undo_dir
 start_server --innodb_file_per_table --innodb_directories="$remote_dir;$undo_dir"
 
 $MYSQL $MYSQL_ARGS test <<EOF
+CREATE TABLESPACE TS1 ADD DATAFILE '$remote_dir/ts1.ibd';
+CREATE TABLESPACE TS2 ADD DATAFILE '$remote_dir/ts2_name_mismatch.ibd';
 CREATE TABLE t(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
   DATA DIRECTORY = '$remote_dir' ENGINE=InnoDB;
 INSERT INTO t(c) VALUES (1), (2), (3), (4), (5), (6), (7), (8);
+CREATE TABLE b(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+  TABLESPACE TS1  ENGINE=InnoDB;
+INSERT INTO b(c) VALUES (1), (2), (3), (4), (5), (6), (7), (8);
+CREATE TABLE c(id INT AUTO_INCREMENT PRIMARY KEY, c INT)
+  TABLESPACE TS2 ENGINE=InnoDB;
+INSERT INTO c(c) VALUES (1), (2), (3), (4), (5), (6), (7), (8);
```

Comment:
but you don't access them after restore and copy-back. So whats the point in adding them?

Reply:
they accessed and compared in verify_db_state.